scss/utilities/_layout.scss

```diff
@@ -0,0 +1,27 @@
+// position utilites
+.position-static   { position: static !important; }
+.position-relative { position: relative !important; }
+.position-absolute { position: absolute !important; }
+.position-fixed    { position: fixed !important; }
+
+.top-0    { top: 0 !important; }
+.right-0  { right: 0 !important; }
+.bottom-0 { bottom: 0 !important; }
+.left-0   { left: 0 !important; }
+
+// Display utilites
+.d-block        { display: block !important; }
+.d-inline       { display: inline !important; }
+.d-inline-block { display: inline-block !important; }
+.hidden, // legacy
+.d-none         { display: none !important; }
+
+
+// Clear floats
+.clearfix {
+  @include clearfix;
+}
+
+// Floats
+.right { float: right !important; }
+.left { float: left !important; }
```

Comment:
these are legacy, we can't rename them yet without a bunch of refactor work. :disappointed: